src/orderbook.js

```diff
@@ -164,7 +201,15 @@ OrderBook.EVENTS = [
   'offer_changed', 'offer_funds_changed'
 ];
 
-OrderBook.DEFAULT_TRANSFER_RATE = 1000000000;
+OrderBook.DEFAULT_TRANSFER_RATE = Amount.from_json(1000000000);
+
+OrderBook.NOTIFY_THROTTLE_TIME = 4000;
```

Comment:
It has to do with why autobridge compute is being called.  Its being called multiple times needlessly , and debouncing it fixes that.  Before any optimizations, if in 1 ledger 5 transactions affected 1 leg, and 3 transactions affected another, and 2 transactions affected the IOU/IOU book, compute and merge would be called 10 times. 

Throttling it to every 5 seconds with trailing set to true in this case will result in compute being called 2 times for that ledger - once at the very first transaction, and then once again 5 seconds later with all the changes from the ledger.  

Debouncing it will (generally) result in notify being called once per leg and IOU/IOU book per ledger, and (generally) compute being called only once per autobridged book per ledger.  And it gets called immediately after *all* the transactions have been applied, rather than 5 seconds later.

Throttling reduces the calls to compute, but debouncing fixes the reasons why compute is being called so often in the first place.